### DISCUSSION TITLE
ユーザー辞書: 単漢字の追加

### DISCUSSION THREAD

**Discussion by ghost** [google/mozc#539] [Q&A]
単漢字変換で候補に表示される漢字を追加したいのですが、ユーザー辞書から追加することは可能でしょうか。
品詞を見た限り、単漢字というのは見当たりませんでした。

**Comment by faketanaka** [comment]
これはぜひとも欲しい機能です。基本的には単漢字についてはイシューなりプルリクエストなりで対応すればいいと思いますが、一般的ではない読みで単漢字変換を行いたい場合、Mozcの内蔵辞書に追加するのが好ましくない場合もあるでしょう。

なお、ATOKにはユーザー辞書の品詞に「単漢字」があります。
